src/foundry/client/pixi/layers/base/placeables-layer.d.mts

```diff
@@ -415,6 +476,8 @@ declare global {
     data: InstanceType<Document.ConfiguredClassForName<DocumentName>>["_source"][];
   }
 
+  type PlaceablesLayerOptions<DocumentName extends Document.PlaceableType> = PlaceablesLayer.LayerOptions<DocumentName>
```

Comment:
What's up with adding this?

Reply:
They extracted the type for `layerOptions` out to a typedef, so I wanted to make sure that typedef was nominally available.